components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/builder/FileBasedConfigurationBuilder.java

```diff
@@ -189,7 +189,10 @@ private void buildConfiguration() {
             log.error(IdentityApplicationConstants.APPLICATION_AUTHENTICATION_CONGIG + " file is not available", e);
         } catch (XMLStreamException e) {
             log.error("Error reading the " + IdentityApplicationConstants.APPLICATION_AUTHENTICATION_CONGIG, e);
-        } finally {
+        } catch (Exception e) {
+            log.error("Syntax error in "+ IdentityApplicationConstants.APPLICATION_AUTHENTICATION_CONGIG, e);
```

Comment:
Is an Exception is thrown only for a syntax error ? Can't there be an Exception for any other error ?